features/roomlist/impl/src/test/kotlin/io/element/android/features/roomlist/impl/RoomListPresenterTests.kt

```diff
@@ -66,7 +69,8 @@ class RoomListPresenterTests {
 
```

Comment:
It doesn't look like the new code was covered by tests.

Reply:
Test added for this case, thanks for the reminder.